frontend-landing/assets/scripts/expansion-card.js

```diff
@@ -0,0 +1,42 @@
+var expansion = {}
+var collapse = {}
+var info = {}
+var social = {}
+var bg = {}
+
+for (var i = 0; i < 25; i++) {
```

Comment:
This is a very, very unsafe way to do this. Select the elements you need with CSS classes and loop over that with (though I _do_ appreciate that they will all show without JavaScript :+1:)

As it stands you are:

1. assuming the list has < 25 members
2. looping unnecessary for list with < 24 members
3. not covering accessibility
4. not leveraging the selector engine--which is very fast--which can get you the precise amount of elements instead of setting some ceiling _and_ allow a reusable selector.

I would read about [`aria-expanded`](https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions). BONUS: If you just use and toggle these `aria-*` attributes, your code will be accessible to screen readers for our users with special needs. An example:

```javascript
var $$ = document.querySelectorAll

// the `.Collapsible` class is a guard against elements that need 
// a different wiring or come from somewhere else.
Array.prototype.forEach.call($$(".Collapsible[aria-expanded]"), function(element) {
  // ... do stuff like set up event handlers
});
```

and as such your CSS should look something like (I don't know your exact styling but you get the idea


```css
.Collapsible[aria-expanded="false"] {
  display: none;
}
```

This should ensure that you _never_ get the styling _unless_ you satisfy the accessibility requirement in `[aria-expanded]`.

As I was telling @CSVdB this week, it's _always_ a good idea to draw from folks' ARIA examples so that a) you're covering for accessibility, b) you don't have to reinvent how to do it or think about it and can 'steal' the accessible example from the source.